src/main/java/io/openliberty/tools/intellij/actions/ViewIntegrationTestReport.java

```diff
@@ -54,13 +53,12 @@ protected void executeLibertyAction(LibertyModule libertyModule) {
 
 
         if (failsafeReportVirtualFile == null || !failsafeReportVirtualFile.exists()) {
-            Notification notif = new Notification(Constants.LIBERTY_DEV_DASHBOARD_ID
-                    , LibertyPluginIcons.libertyIcon
-                    , LocalizedResourceUtil.getMessage("integration.test.report.does.not.exist.notification.title")
-                    , ""
-                    , LocalizedResourceUtil.getMessage("test.report.does.not.exist", failsafeReportFile.getAbsolutePath())
-                    , NotificationType.ERROR
-                    , NotificationListener.URL_OPENING_LISTENER);
+            Notification notif = new Notification(Constants.LIBERTY_DEV_DASHBOARD_ID,
```

Comment:
@aparnamichael , How many parameters can this method `Notification` actually have?
I can see the change in the number of parameters in other usages of same method `Notification ` , So Can you confirm , the changes are ok?

Reply:
There are 2 recommended constructors, as I mentioned in the above comment. One with 3 parameter and another with 4 parameter. ie, the constructor with 3 parameter doesn't have an option to add title. We have title for all the notifications so I I only used the constructor with 4 parameters every where.

```
public Notification(     @NotNull String groupId,
    @NotNull String title,
    @NotNull String content,
    @NotNull NotificationType type )
```